projects/admin/src/app/pipe/get_record.pipe.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { RecordService } from '@rero/ng-core';
+import { map } from 'rxjs/operators';
+
+@Pipe({
+  name: 'getRecord'
+})
+export class GetRecordPipe implements PipeTransform {
```

Comment:
GetRecordPipe is integrated in ng-core, I think you can remove this.

Reply:
done